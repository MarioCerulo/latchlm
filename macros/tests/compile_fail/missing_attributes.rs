#![allow(unused)]

use latchlm_core::AiModel;
use latchlm_macros::AiModel;

#[derive(AiModel)]
enum Model {
    Variant,
}

fn main() {}
