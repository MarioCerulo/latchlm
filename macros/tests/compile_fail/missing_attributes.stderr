error: missing #[model] attribute with id
 --> tests/compile_fail/missing_attributes.rs:8:5
  |
8 |     Variant,
  |     ^^^^^^^
